stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/viewholder/internal/TextAndAttachmentsViewHolder.kt

```diff
@@ -75,7 +75,10 @@ internal class TextAndAttachmentsViewHolder(
         binding.messageText.isVisible = data.message.text.isNotEmpty()
         markdown.setText(binding.messageText, data.message.text)
 
-        setupAttachment(data)
+        if (diff?.attachments == true) {
```

Comment:
Seems like if diff is null we don't draw attachments. Is it right? Maybe we can make diff not-nullable passing some default empty value for example

Reply:
The default empty value wouldn't be the original/correct data, which can be confusing. Fixing the nullability of the diff is outside of the scope of this PR. Let's create another PR an investigate why the diff can be null. 

About the draw with null diff, what about drawing it if the diff is null and only not drawing when it is false? This is closer from the current behaviour